src/Components/profile/Profile.js

```diff
@@ -29,18 +29,19 @@ const Profile = () => {
   const [emailUser, setEmailUser] = useState("");
   const [seeModal, setSeeModal] = useState(false);
   const [signOut, setSignOut] = useState(false);
-  const [eventClicked, setEventClicked] = useState("");
+  const [id, setId] = useState("");
   const [goToDashboard, setGoToDashboard] = useState(false);
   const [goToDetailEvent, setGoToDetailEvent] = useState(false);
   const [goToEditEvent, setGoToEditEvent] = useState(false);
   const [eventToEdit, setEventToEdit] = useState("");
   const eventsFromLocalStorage = getEventsFromLocalStorage(pageNumber);
-  const [eventsList, setEventList] = useState(eventsFromLocalStorage.events);
```

Comment:
we can keep this code but initialize it on empty:
const [eventsList, setEventList] = useState([]);